up.go

```diff
@@ -175,8 +183,28 @@ func main() {
 		drawText(TuiRegion(tui, 0, h-1, w, 1), whiteOnBlue, message)
 		tui.Show()
 
+		var autoRunTimer *time.Timer
+		if command != lastCommand && autoRunDelay > 0 {
+			autoRunTimer = time.AfterFunc(autoRunDelay, func() {
+				log.Println("Firing interrupt - poll timed out")
+				tui.PostEvent(tcell.NewEventInterrupt("autoRunTimer"))
```

Comment:
Mwahaha! I just realized we can probably do a sneaky hack here, by replacing NewEventInterrupt with `NewEventKey(tcell.KeyEnter, ...)`; this should let us remove the whole l.201-207 block!